src/summary/metadata/SarcomaSummarySection.jsx

```diff
@@ -172,32 +171,32 @@ export default class SarcomaSummarySection extends MetadataSection {
                             },
                             // shortcut: "@stage"
                         },
-                        {
-                            name: "Mitosis",
-                            value: (patient, currentConditionEntry) => {
-                                let mr = currentConditionEntry.getMostRecentMitosis();
-                                if (mr) {
-                                    return  {   value: mr.quantity.number + " " + mr.quantity.unit, 
-                                                isUnsigned: patient.isUnsigned(mr), 
-                                                source: this.determineSource(patient, mr)
-                                            };
-                                } else {
-                                    return null;
-                                }
-                            },
-                        },
-                        {
-                            name: "Tumor Size",
-                            value: (patient, currentConditionEntry) => {
-                                const list = currentConditionEntry.getObservationsOfTypeChronologicalOrder(FluxTumorDimensions);
-                                if (list.length === 0) return null;
-                                const size = list.pop(); // last is most recent
-                                return  {   value: size.quantity.value + " " + size.quantity.unit, 
-                                            isUnsigned: patient.isUnsigned(size), 
-                                            source: this.determineSource(patient, size)
-                                        };
-                    },
-                        },
+                        // {
```

Comment:
SWRT 174-199? Also, looks like there was a justification for removing tumor and mitotic division rate, could you talk to that a bit?

Reply:
The tumor and mitotic division rate got moved to recent lab results.  We didn't want duplicate information in the summary section so we removed it.

Reply:
Gotcha!